sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -177,12 +178,12 @@ class JDBCSuite extends QueryTest
        """.stripMargin.replaceAll("\n", " "))
 
     conn.prepareStatement("CREATE TABLE test.timezone (tz TIMESTAMP WITH TIME ZONE) " +
-      "AS SELECT '1999-01-08 04:05:06.543543543 GMT-08:00'")
+      "AS SELECT '1999-01-08 04:05:06.543543543-08:00'")
       .executeUpdate()
     conn.commit()
 
-    conn.prepareStatement("CREATE TABLE test.array (ar ARRAY) " +
-      "AS SELECT '(1, 2, 3)'")
+    conn.prepareStatement("CREATE TABLE test.array_table (ar INTEGER ARRAY) " +
+      "AS SELECT ARRAY[1, 2, 3]")
```

Comment:
1. why need rename `test.array` -> `test.array_table` ? Because Array standard a data type in newer H2. So H2 throws:
```
An exception or error caused a run to abort: Syntax error in SQL statement "CREATE TABLE TEST.ARRAY[*] (AR INTEGER ARRAY) AS SELECT ARRAY[1, 2, 3]"; expected "identifier"; SQL statement:
CREATE TABLE test.array (ar Integer ARRAY) AS SELECT ARRAY[1, 2, 3] [42001-202] 
```
2. Why need add `INTEGER` ? Because ARRAY need identify data type in newer H2. otherwise:
```
An exception or error caused a run to abort: Syntax error in SQL statement "CREATE TABLE TEST.ARRAY_TABLE (AR ARRAY[*]) AS SELECT ARRAY[1, 2, 3]"; expected "IDENTITY, data type"; SQL statement:
CREATE TABLE test.array_table (ar ARRAY) AS SELECT ARRAY[1, 2, 3] [42001-202] 
```
3. Why need change `'(1, 2, 3)'`  to `ARRAY[1, 2, 3]`? Because cannot convert it to ARRAY in newer H2.
```
An exception or error caused a run to abort: Data conversion error converting "'(1, 2, 3)' (ARRAY_TABLE: ""AR"" INTEGER ARRAY)"; SQL statement:
CREATE TABLE test.array_table (ar Integer ARRAY) AS SELECT '(1, 2, 3)' [22018-202] 
```